platform/ui/src/contextProviders/ViewportGridProvider.jsx

```diff
@@ -27,14 +27,10 @@ export function ViewportGridProvider({ children, service }) {
         return { ...state, ...{ activeViewportIndex: action.payload } };
       }
       case 'SET_DISPLAYSET_FOR_VIEWPORT': {
-        const {
-          viewportIndex,
-          displaySetInstanceUID,
-          imageIndex,
-        } = action.payload;
+        const { viewportIndex, displaySetInstanceUID } = action.payload;
```

Comment:
I believe this becomes an API change that needs to be reflected in the `ViewportGridService` in @ohif/core

Reply:
Yes, that is the point of the PR, we removed imageIndex throughout, and it affects the `ViewportGridProvider` and thus the `ViewportGridService`, unless I'm missing something?

Reply:
`platform/core/src/srvices/ViewportGridService/ViewportGridService.js`
--> `_setDisplaysetForViewport`

Defines the interface for the service. I didn't see it updated in this file. It indicates `viewportIndex` as an expected property for the method.